client/python/armada_client/permissions.py

```diff
@@ -20,6 +20,17 @@ def to_grpc(self) -> submit_pb2.Queue.Permissions.Subject:
 class Permissions:
     """
     Permissions including Subjects and Verbs
+
+    Note that to_grpc() is not needed to use the client module:
```

Comment:
I don't understand the implications of this - is something else in the docs hinting that a to_grpc() func might be needed?